Domain/Models/Office.cs

```diff
@@ -9,6 +9,8 @@ namespace Domain.Models
     public class Office : BaseEntity
     {
         public string Name { get; set; }
+        public string Country { get; set; }
+        public string City { get; set; }
```

Comment:
The same with Country

Reply:
I think for now we are OK with the simplest model.